### PR TITLE
add vscode mentions

### PR DESCRIPTION
**Pull request by noklam**
## Description

Partly: https://github.com/kedro-org/vscode-kedro/issues/11
Add mentions of VSCode extension.
## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

File: docs/source/development/set_up_vscode.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ At this stage, you should be able to see the `conda` environment that you have c
 
 ![](../meta/images/vscode_setup_interpreter.png)
 
+## Kedro VSCode Extension
+[Kedro VSCode extension](https://marketplace.visualstudio.com/items?itemName=kedro.Kedro) support 0.19+ versions of Kedro project. It supports features like go-to definition, find references and hovers etc.
```
Comment (astrojuanlu) [inline docs/source/development/set_up_vscode.md:14]: Just for marketing purposes, can we include one of the animated GIFs? 😇

File: docs/source/development/set_up_vscode.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ At this stage, you should be able to see the `conda` environment that you have c
...
```
Comment (astrojuanlu) [inline docs/source/development/set_up_vscode.md:14]:
```suggestion
[Kedro VSCode extension](https://marketplace.visualstudio.com/items?itemName=kedro.Kedro) supports Kedro 0.19+. It helps you navigate around your Kedro project by finding the definition of your datasets, find references to them in your code, and more.
```

Wdyt @stichbury?

File: docs/source/get_started/install.md [hunk 3]
```diff
@@ -80,7 +80,11 @@ To exit `kedro-environment`:
 ```bash
 conda deactivate
 ```
+### Optional: Setup Kedro with IDE
```
Comment (astrojuanlu) [inline docs/source/get_started/install.md:83]:
```suggestion
### Optional: Add Kedro support to your IDE
```

? (Unusure about this one as well, requesting @stichbury's opinion)

File: docs/source/get_started/install.md [hunk 4]
```diff
@@ -80,7 +80,11 @@ To exit `kedro-environment`:
...
+Working in IDEs can be a great productivity boost.
```
Comment (astrojuanlu) [inline docs/source/get_started/install.md:84]:
```suggestion
Working in an IDE can be a great productivity boost.
```

**Review by astrojuanlu — COMMENTED** [review]
Left some comments

File: docs/source/development/set_up_vscode.md [hunk 5]
```diff
@@ -10,6 +10,10 @@ At this stage, you should be able to see the `conda` environment that you have c
 
 ![](../meta/images/vscode_setup_interpreter.png)
 
+## Kedro VSCode Extension
```
Comment (merelcht) [inline docs/source/development/set_up_vscode.md:13]: Nit: in most places you refer to `VS Code` with a space but here it's `VSCode`, probably best to pick one for consistency

File: docs/source/get_started/install.md (hunk 3 above)
Comment (noklam) [reply docs/source/get_started/install.md:83]: > [Integrate ESLint in VSCode with the official extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)

Integrate Kedro in VS Code with the official extension? @astrojuanlu

Comment (astrojuanlu) [reply docs/source/get_started/install.md:83]: +1!

**Review by merelcht — APPROVED** [review]

**Comment by astrojuanlu** [comment]
```
sphinx.errors.SphinxWarning: /home/docs/checkouts/readthedocs.org/user_builds/kedro/checkouts/3898/docs/source/get_started/install.md:86:local id not found in doc 'development/set_up_vscode': 'kedro-vscode-extension' [myst.xref_missing]

Warning, treated as error:
/home/docs/checkouts/readthedocs.org/user_builds/kedro/checkouts/3898/docs/source/get_started/install.md:86:local id not found in doc 'development/set_up_vscode': 'kedro-vscode-extension' [myst.xref_missing]
```

File: docs/source/get_started/install.md [hunk 6]
```diff
@@ -80,7 +80,11 @@ To exit `kedro-environment`:
 ```bash
 conda deactivate
 ```
+### Optional: Setup Kedro with IDE
+Working in an IDE can be a great productivity boost.
 
+For VS Code Users: Checkout [Set up Visual Studio Code](../development/set_up_vscode.md) and [Kedro VS Code Extension](../development/set_up_vscode.md#kedro-vscode-extension)
```
Comment (astrojuanlu) [inline docs/source/get_started/install.md:86]: I guess

```suggestion
For VS Code Users: Checkout [Set up Visual Studio Code](../development/set_up_vscode.md) and [Kedro VS Code Extension](../development/set_up_vscode.md#kedro-vs-code-extension)
```

?

**Pull request merged**